packages/apollo/src/commands/service/check.ts

```diff
@@ -193,15 +357,17 @@ export default class ServiceCheck extends ProjectCommand {
       );
     }
 
-    const {
-      targetUrl,
-      diffToPrevious: { changes, validationConfig }
-    } = checkSchemaResult;
-    const failures = changes.filter(({ type }) => type === ChangeType.FAILURE);
-
     if (shouldOutputJson) {
       return this.log(
-        JSON.stringify({ targetUrl, changes, validationConfig }, null, 2)
+        JSON.stringify(
+          {
+            targetUrl: checkSchemaResult,
```

Comment:
Is this line correct? Looks like it's getting everything instead of just a relevant slice of the data.

Reply:
Great catch!